packages/editor/src/components/post-schedule/label.js

```diff
@@ -2,14 +2,14 @@
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { dateI18n, __experimentalGetSettings } from '@wordpress/date';
+import { format, __experimentalGetSettings } from '@wordpress/date';
 import { withSelect } from '@wordpress/data';
 
 export function PostScheduleLabel( { date, isFloating } ) {
 	const settings = __experimentalGetSettings();
 
 	return date && ! isFloating
-		? dateI18n(
```

Comment:
It seems the intent of that function is not clear. Does it assume that the passed date is a given timezone? Should we remove that assumption? Should we clarify it in the description of the function.

Anyway, it seems like the current fix is good anyway but there might be something wrong with `dateI18n`

Reply:
I think by default the dateI18n assumes the time is in the user's timezone and formats it in the website time zone.
I think we may have other cases where we are using dateI18n for dates that are already on the website timezone. I will try to propose a follow-up PR that improves documentation for this function and maybe fixes additional issues.